### PR TITLE
7 feature follow

### PR DESCRIPTION
**Pull request by LFTPadilla**

### CODE REVIEW

File: test/fixtures/relationships.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```
Comment (MrJuanGaviriaK) [inline test/fixtures/relationships.yml:3]: We don't need unused files!

File: test/models/relationship_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (MrJuanGaviriaK) [inline test/models/relationship_test.rb:1]: We don't need unused files!

**Review by MrJuanGaviriaK — CHANGES_REQUESTED** [review]
Hi @LFTPadilla 

I left some minor changes!!! 

Bests,

File: test/fixtures/relationships.yml (hunk 1 above)
Comment (LFTPadilla) [reply test/fixtures/relationships.yml:3]: Ups, I forget it.

File: config/routes.rb [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
   devise_scope :user do
     get '/users/sign_out' => 'devise/sessions#destroy'
-    get '/users/user_url' => 'article#index'
+    get '/users/user_url' => 'articles'
   end
   resources :articles do
     resources :comments
   end
+  resources 'users' do
+    get 'follow/:id_article', to: 'users#new_follower', as: 'follow'
```
Comment (MrJuanGaviriaK) [inline config/routes.rb:13]: Do you need the article_id in these routes??

Comment (LFTPadilla) [reply config/routes.rb:13]: Yes, I use it to redirect to the article. Example: If I follow the author from this article, it redirects to the same article.
image

**Review by MrJuanGaviriaK — APPROVED** [review]

**Pull request merged**